website/content/v1.7/kubernetes-guides/network/deploying-cilium.md

```diff
@@ -63,31 +63,31 @@ Install the [Cilium CLI](https://docs.cilium.io/en/v1.13/gettingstarted/k8s-inst
 
 ```bash
 cilium install \
-    --helm-set=ipam.mode=kubernetes \
-    --helm-set=kubeProxyReplacement=disabled \
-    --helm-set=securityContext.capabilities.ciliumAgent="{CHOWN,KILL,NET_ADMIN,NET_RAW,IPC_LOCK,SYS_ADMIN,SYS_RESOURCE,DAC_OVERRIDE,FOWNER,SETGID,SETUID}" \
-    --helm-set=securityContext.capabilities.cleanCiliumState="{NET_ADMIN,SYS_ADMIN,SYS_RESOURCE}" \
-    --helm-set=cgroup.autoMount.enabled=false \
-    --helm-set=cgroup.hostRoot=/sys/fs/cgroup
+    --set ipam.mode=kubernetes \
+    --set kubeProxyReplacement=disabled \
+    --set securityContext.capabilities.ciliumAgent={CHOWN,KILL,NET_ADMIN,NET_RAW,IPC_LOCK,SYS_ADMIN,SYS_RESOURCE,DAC_OVERRIDE,FOWNER,SETGID,SETUID} \
```

Comment:
I think the quotes `"` should stay, as `{` might be a shell special symbol

Reply:
Updated it in all locations. Thanks for this feedback. Even though it worked in my shell without quotes I did some reading and learned it might break in some shells.